### ISSUE TITLE
Play with hyperparameters (In GPU)

### ISSUE THREAD

**Issue by shayanaimi** [medvidov/IMaSC#66]
Is there a way to make this issue like "dependent" on another issue? The GPU one.
Anyway, running performance_vs_training on the GPU and using plotly will enable us to tune hyperparameters and quickly visualize results using 3D plots and stuff. I'd like to tune: 

- [ ] Learning rate 
- [ ] beta1 and beta2
- [ ] Dropout rate (it's currently .35 but should be .1-.25)
- [ ] Maybe make this a new issue, but use plotly to visualize the results of the hyperparameters
